Signal/src/view controllers/OWSConversationSettingsTableViewController.m

```diff
@@ -211,6 +211,10 @@ - (void)viewDidLoad
 - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
+
+    // Since we're using a custom back button, we have to do some extra work to manage the interactivePopGestureRecognizer
+    self.navigationController.interactivePopGestureRecognizer.delegate = self;
```

Comment:
I think the reason I added it everywhere was that:

* It looks really bad if you transition from a VC "default" back button to a "custom" back button and vice versa.  So we need to use the "custom" back button in any view which can be reached from the a view using the "custom" back button.
* We need the "custom" back button in the messages VC so that we can use the left-aligned title.
* You can reach the "scan QR code" view from the messages VC _and_ the settings VC. 
* Therefore we need to use the "custom" back button in 90-100% of our VCs, so I just decided to use it everywhere.  

I'm very open to suggesting another approach.  We could, for example, add a "useCustomBackButton" property to the "scan QR code" view.